pandas/tests/extension/category/test_categorical.py

```diff
@@ -75,6 +75,10 @@ def test_align(self, data, na_value):
     def test_align_frame(self, data, na_value):
         pass
 
+    @pytest.mark.skip(reason="Unobserved categories preseved in concat.")
+    def test_align_series_frame(self, data, na_value):
```

Comment:
Isn't this only because the way the test is constructed? You can also do it with the full of `data` instead of only the first 3 elements. Then this is not an issue?